commands.go

```diff
@@ -111,8 +130,12 @@ func (cmd Command) Run(label string, cmdline ...string) error {
 			options = append(options, "--bind", b)
 
 		}
-		options = append(options, cmdline...)
 	}
+	if len(cmdline) == 1 {
+		// Single command OR command with options passed as single string
+		options = append(options, "sh", "-c")
```

Comment:
I don't think this class should try to guess if it should use sh -c, the caller should do that.